### DISCUSSION TITLE
网站不存在，请检查域名[BUG]

### DISCUSSION THREAD

**Discussion by czqasngit** [1Panel-dev/1Panel#5418] [Q&A]
### 1Panel 版本

v1.10.5-lts

### 问题描述

访问1panel管理面板提示"网络不存在，请检查域名"
![image](https://github.com/1Panel-dev/1Panel/assets/15681141/bbbd648d-d87a-462c-9263-506d0cc177a0)

### 重现步骤

过了几天没访问就这样了，不知道怎么复现

**Comment by zhengkunwang223** [comment]
访问的域名在网站列表中吗？

**Comment by czqasngit** [comment]
> 访问的域名在网站列表中吗？

我是使用IP地址直接访问的，http://47.120.58.228/

**Comment by zhengkunwang223** [comment]
> > 访问的域名在网站列表中吗？
> 
> 我是使用IP地址直接访问的，http://47.120.58.228/

无论你是什么访问 只要这个地址不在网站列表 就会拦截

**Reply by czqasngit** [reply]
> > > 访问的域名在网站列表中吗？
> > 
> > 
> > 我是使用IP地址直接访问的，http://47.120.58.228/
> 
> 无论你是什么访问 只要这个地址不在网站列表 就会拦截

我现在已经进不去控制面板了，我应该怎么去处理这个问题，或者我需要怎么配置就可以进入到面板访问了？上面我给出的这个地址是1panel的控制面板

**Comment by czqasngit** [comment]
那我现在应该怎么去处理问题呢？

**Reply by zhengkunwang223** [reply]
关闭waf即可

**Reply by czqasngit** [reply]
我现在进不去面板页面了，使用命令行怎么关闭waf？

**Reply by zhengkunwang223** [reply]
重启 openresty 即可
